components/nexus-bootstrap/src/main/java/org/sonatype/nexus/bootstrap/jetty/InstrumentedSelectChannelConnector.java

```diff
@@ -1,99 +0,0 @@
-/*
```

Comment:
Yes. This is still wip, more changes to go.

thanks (phone),
~t~
On Jan 22, 2015 3:52 PM, "Stuart McCulloch" <notifications@github.com>
wrote:

> In
> components/nexus-bootstrap/src/main/java/org/sonatype/nexus/bootstrap/jetty/InstrumentedSelectChannelConnector.java
> <https://github.com/sonatype/nexus-oss/pull/989#discussion_r23379730>:
>
> > @@ -1,99 +0,0 @@
> > -/*
>
> Both InstrumentedSelectChannelConnector and
> InstrumentedSslSelectChannelConnector are referenced in etc/jetty.xml and
> etc/jetty-https.xml - so those files will need updating (to presumably
> use the new InstrumentedConnectionFactory?).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sonatype/nexus-oss/pull/989/files#r23379730>.
>